LibreHardwareMonitor/UI/MainForm.cs

```diff
@@ -525,6 +589,29 @@ private void InitializePlotForm()
             treeView.Invalidate();
         };
 
+        _strokeThickness = new UserRadioGroup("plotStroke", 1, new[] { strokeThickness1ptMenuItem, strokeThickness2ptMenuItem, strokeThickness3ptMenuItem, strokeThickness4ptMenuItem }, _settings);
+
+        _strokeThickness.Changed += (sender, e) =>
+        {
+            switch (_strokeThickness.Value)
```

Comment:
What about a default safe result if the value falls outside of the switch bounds?

You could also simplify this by a simple

```csharp
_plotStrokeThickness = (_strokeThickness.Value >= 0 && _strokeThickness.Value <=3)
                                                   ? _strokeThickness.Value + 1
                                                   : 4; // Or other safe value
```

Reply:
Implemented as suggested